app/main/views/briefs.py

```diff
@@ -360,12 +358,7 @@ def view_response_result(brief_id):
         supplier_id=current_user.supplier_id
     )['briefResponses']
 
-    legacy_brief = True
-    if current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW']:
-        legacy_brief = (
-            datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
-            > datetime.strptime(brief['publishedAt'], DATETIME_FORMAT))
-
+    legacy_brief = not new_supplier_flow_active_for_datetime(brief_published_at)
     if len(brief_response) == 0:
```

Comment:
Resolves to:
`(not feature_flag_date <= brief_published_at)`
which is equal to:
` feature_flag_date > brief_published_at`